go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bxcodec/dbresolver
 
-go 1.17
+go 1.19
```

Comment:
why does this change exist?
This will be a breaking change. Because it will require at least go 1.19 to be able to use this library

Reply:
Go 1.19 is backwards compatible

Reply:
thanks to [Go 1 compatibility promise](https://click.convertkit-mail2.com/n4u9zol3zlfvhmqnvpf6/e0hph7hk9rqmzqb8/aHR0cHM6Ly9nb2xhbmcub3JnL2RvYy9nbzFjb21wYXQ=).

Reply:
It's backward compatible if I'm using a newer one and the libraries are still using the old one, I can still use the libraries. 
But it won't work if the library is the newer one, but my project uses the older one. I won't be able to use it 🤔 

Need to confirm tho 🤔

Reply:
I mean, if we really don't need a new feature from go 1.19, it's better to keep it on go 1.17

eg, if we use Generic (go 1.18) then we have to make sure the minimum version in our Go mod is Go 1.18. But we're not using generic here.